MarsAdvancedTaskPart1NUnitAutomationV3/Steps/LanguageSteps.cs

```diff
@@ -0,0 +1,253 @@
+﻿using MarsAdvancedTaskPart1NUnitAutomation.AssertHelpers;
+using MarsAdvancedTaskPart1NUnitAutomation.DataModel;
+using MarsAdvancedTaskPart1NUnitAutomation.Pages.ProfileOverview.ProfileNavigationMenuComponents;
+using MarsAdvancedTaskPart1NUnitAutomation.Pages.ProfileOverview.ProfileNavigationMenuComponents.RenderingComponents;
+using MarsAdvancedTaskPart1NUnitAutomation.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MarsAdvancedTaskPart1NUnitAutomation.Steps
+{
+
+    public class LanguageSteps : CommonDriver
+    {
+        string actualMessage = string.Empty;
+        string expectedMessage = string.Empty;
+        string languageName = string.Empty;
+
+
+        int cancelFlag = 0;
+        LanguageAddAndUpdateComponent languageAddAndUpdateComponent = new LanguageAddAndUpdateComponent();
+
+
+        //To capture the pop up message
+        public void CapturePopupMessage()
+        {
+
+            Thread.Sleep(1000);
+
+            IWebElement messageBox = driver.FindElement(By.XPath("//div[@class='ns-box-inner']"));
```

Comment:
all web elements should be keep in pages as this is a page logic and not step, you should create methods within the correct classes and return the actual method! Also the actual message should be returned so that is use not just used for WriteLine as it does nothing! please fix this everywhere